<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewWidget</class>
 <widget class="QWidget" name="PreviewWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>341</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>328</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preview</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonPreviousFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../ressources.qrc">
         <normaloff>:/images/images/previous_frame.png</normaloff>:/images/images/previous_frame.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPlay">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../ressources.qrc">
         <normaloff>:/images/play</normaloff>:/images/play</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNextFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../ressources.qrc">
         <normaloff>:/images/images/next_frame.png</normaloff>:/images/images/next_frame.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMarkerStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../ressources.qrc">
         <normaloff>:/images/marker_start</normaloff>:/images/marker_start</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMarkerStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../ressources.qrc">
         <normaloff>:/images/marker_stop</normaloff>:/images/marker_stop</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCreateClip">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../ressources.qrc">
         <normaloff>:/images/editcut</normaloff>:/images/editcut</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../ressources.qrc">
         <normaloff>:/images/images/stop.png</normaloff>:/images/images/stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LCDTimecode" name="lcdNumber">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="numDigits">
        <number>11</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="renderWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="previewLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="PreviewRuler" name="rulerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewRuler</class>
   <extends>QWidget</extends>
   <header>PreviewRuler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LCDTimecode</class>
   <extends>QLCDNumber</extends>
   <header>LCDTimecode.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
